components/tor/tor_profile_service_impl.cc

```diff
@@ -148,6 +166,29 @@ void TorProfileServiceImpl::OnExecutableReady(const base::FilePath& path) {
   }
 }
 
+void TorProfileServiceImpl::OnPluggableTransportReady(bool success) {
+  if (!success || !tor_launcher_factory_)
+    return;
+  OnBridgesConfigChanged();
+}
+
+void TorProfileServiceImpl::OnBridgesConfigChanged() {
+  auto config = tor::BridgesConfig::FromValue(
+                    local_state_->GetDictionary(tor::prefs::kBridgesConfig))
+                    .value_or(tor::BridgesConfig());
+  if (!tor_pluggable_transport_updater_ ||
```

Comment:
`!tor_pluggable_transport_updater_` should just return otherwise it is likely to crash later doing `tor_pluggable_transport_updater_->Register()`